reactor-core/src/test/java/reactor/core/publisher/scenarios/FluxTests.java

```diff
@@ -1591,6 +1593,48 @@ public void splitBugEventuallyHappens() throws Exception {
 
 	}
 
+	@Test
+	public void fluxFromCallsAssemblyHook() {
```

Comment:
WDYT about splitting it into multiple tests (maybe with some parameterization)? 
It will be a bit weird when we break one to not know which one exactly :)

Reply:
I could split and do the whole `onEachOperator` ceremony in multiple shorter tests, but note that we'll know which one exactly thanks to the `as` description of each assert...

Reply:
btw `resetOnEachOperator` can to go `@AfterTest`, so that the test becomes a 2-3 liner

`as` is nice, but I have bad memory about debugging sequential tests (at least without soft assertions) 😅